tests/tpubkey

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash -e
+# Copyright (C) 2022 Jakub Jelen <jjelen@redhat.com>
+# SPDX-License-Identifier: Apache-2.0
+
+source ${TESTSSRCDIR}/helpers.sh
+
+TSTOUT="${TMPPDIR}/testout"
```

Comment:
Btw this is not ok, it makes copy/pasting command to run on the command line use an undefined variable which means the manual test I ran crated a file named "-priv-cert.pub" in the current directory.

Reply:
Just use ${TMPDIR}, non need for a separate directory for outputs...

Reply:
This was just moved from `tests/tbasic`. But fine to remove that.

Reply:
Ah, then we should fix tests/tbasic as well

Reply:
This PR moves this part of the code from the `tbasic` to `tpubkey`